torchao/_models/sam2/configs/sam2/sam2_hiera_b+.yaml

```diff
@@ -2,18 +2,18 @@
 
 # Model
 model:
-  _target_: sam2.modeling.sam2_base.SAM2Base
+  _target_: torchao._models.sam2.modeling.sam2_base.SAM2Base
```

Comment:
Ah, these are the sam2 configs. There's sam2 and then there's sam2.1. I had to switched to sam2.1, but maybe sam2 is still useful? I updated the sam2.1 configs in https://github.com/pytorch/ao/commit/b2e42ff6b4ed0134684932710a45241910de5fb8#diff-12caf984b30e7028df7fb867a984c5ccad3fde0a0783df2af26db9cef860fbe5 , but in any case it doesn't hurt to have sam2 support as well.